truss/templates/server/common/truss_server.py

```diff
@@ -278,6 +278,24 @@ def create_application(self):
             },
         )
 
+        async def intercept_request_id(request, call_next):
+            request_id = request.headers.get("x-baseten-request-id", None)
+
+            with loguru_logger.contextualize(
+                request_id=request_id, lifecycle=Lifecycle.REQUEST
+            ):
+                response = await call_next(request)
+
+                # don't log health checks
+                is_health_check = request.method == "GET" and (
+                    request.url.path != "/" or request.url.path != "/v1/models/model"
+                )
+                if not is_health_check:
```

Comment:
could we have a separate middleware for doing the access logging (called "def access_logging")? And then change the logic to be as follows:
* Have ACCESS_LOGGING_IGNORE_PATHS be a constant list at the top of this file, where you enumerate the paths that you want to skip
* Use that to determine whether or we should write the access logs or not.